collect-mkfs-confs.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -eu
+
+mkdir -p ./tmp/pkg
+
+WORKDIR=$(pwd)
+# SNAPCRAFT_PART_INSTALL=$(pwd)/snap
```

Comment:
Is this a comment or some left-over? Should it be removed? (or if a comment is a bit confusing)

Reply:
It was used to help me run this locally. I cleaned it.